console/src/App/App.js

```diff
@@ -212,6 +220,12 @@ const AppBase = kind({
 				>
 					{`Popup for ${skinName} skin`}
 				</Popup>
+				<Popup
+					onClose={onToggleDestinationReachedPopup}
+					open={showDestinationReachedPopup}
+				>
+					Destination reached!
```

Comment:
nit: sounds a bit formal maybe? I'm okay as is, but wanted to leave a comment.

Reply:
I agree. This seemed the "safest" in terms of the demo. 🤷‍♂️ Open to suggestions though.